collections/collections.js

```diff
@@ -352,6 +352,10 @@ Schema.ContextBlocks = new SimpleSchema({
   addedAt: {
     type: Date
   },
+  videoMarker: {
+    type: String,
```

Comment:
As its a floating point number I thought string might be the best way to store this and then convert on the fly with parseFloat